src/oscar/apps/offer/abstract_models.py

```diff
@@ -655,7 +655,7 @@ def can_apply_benefit(self, line):
         """
         Determines whether the benefit can be applied to a given basket line
         """
-        return line.stockrecord and line.product.is_discountable
+        return line.stockrecord and line.product.get_is_discountable()
```

Comment:
Could you please add a test demonstrating the issue you are describing?

Reply:
@specialunderwear 

![image](https://user-images.githubusercontent.com/9887968/225201842-64146214-b105-49fa-998d-44b76f9bc978.png)
![image](https://user-images.githubusercontent.com/9887968/225202204-c18f51bd-1e5e-4846-adfd-082fc1df2c1e.png)